image.go

```diff
@@ -0,0 +1,9 @@
+package containerd
+
+import "github.com/containerd/containerd/images"
+
+type Image struct {
+	client *Client
+
+	i images.Image
+}
```

Comment:
@stevvooe Based on the discussion, would it help to introduce a `Ready` function. Or else `Config` or `Manifest` may return weird error during the image is being downloaded or unpacked.